.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,101 @@
+name: ci
```

Comment:
Also need to change the badge and its Chinese version:
https://github.com/apache/apisix/blob/151d74fd9c835d9d1dc5cc7b25fb9bdbf9fc56cf/README.md#L22